### PR TITLE
Minor: Add 2.5 versions

### PR DESCRIPTION
**Pull request by mumrah**
As part of the 2.5.0 release, add the 2.5 versions to various places in the build

### CODE REVIEW

File: tests/docker/Dockerfile [hunk 1]
```diff
@@ -58,6 +58,7 @@ RUN mkdir -p "/opt/kafka-2.1.1" && chmod a+rw /opt/kafka-2.1.1 && curl -s "$KAFK
 RUN mkdir -p "/opt/kafka-2.2.2" && chmod a+rw /opt/kafka-2.2.2 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.2.2.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.2.2"
 RUN mkdir -p "/opt/kafka-2.3.1" && chmod a+rw /opt/kafka-2.3.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.3.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.3.1"
 RUN mkdir -p "/opt/kafka-2.4.1" && chmod a+rw /opt/kafka-2.4.1 && curl -s "$KAFKA_MIRROR/kafka_2.12-2.4.1.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.4.1"
+RUN mkdir -p "/opt/kafka-2.5.0" && chmod a+rw /opt/kafka-2.5.0 && curl -s "$KAFKA_MIRROR/kafka_2.13-2.5.0.tgz" | tar xz --strip-components=1 -C "/opt/kafka-2.5.0"
```
Comment (mumrah) [inline tests/docker/Dockerfile]: Do we want Scala 2.12 or 2.13 here?

Comment (ijuma) [reply tests/docker/Dockerfile]: Let's stick with 2.12 for now.

**Review by omkreddy — APPROVED** [review]
@mumrah Thanks for the PR. LGTM.

**Comment by abbccdda** [comment]
@mumrah Thanks for the PR, we already had a PR to add this: https://github.com/apache/kafka/pull/8378/

**Comment by mumrah** [comment]
@abbccdda let's go with your PR as it includes updates to the actual tests as well. Thanks!

**Pull request closed without merging**